index.html

```diff
@@ -517,6 +517,15 @@ <h3>Archwilio Sir Benfro o sawl safbwynt.</h3>
     g("curtain").remove();
 }, 1000);
 </script>
+
+<script>
```

Comment:
Would it work to shift this bit of script into util.js where a lot of things are initialized? - And keep index.html largely for html. 
I know there's some script in index.html, but it's mostly there just in case the scripts don't load properly.

(At some stage I'd like to simplify it, Instead of all the various hidden pop-ups, it would be better to generate them from JS. This is where component plug-in systems like WordPress do a lot better - all the HTML and CSS and JS to do with any particular feature is in one package, rather than having all this miscellaneous stuff.)